packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -143,16 +136,35 @@ export const unzipAction: GeneratorAction = {
     if (!context.zip) {
       throw new MissKeyError("zip");
     }
-    context.outputs = await unzip(
-      context.zip,
-      context.destination,
-      context.fileNameReplaceFn,
-      context.fileDataReplaceFn,
-      context.filterFn
-    );
+    const tryLimits = 2;
```

Comment:
Let's talk about the design offline since this change break the overall design of downloading actions. It violate the rule of single responsibility for this action and make it not be reused anymore.